### PR TITLE
Module 1: project deployment

### PR DESCRIPTION
**Pull request by dunnyk**
In this branch:
- create a branch project-deployment.
- Update README.md file to appear professional.
- Do pull request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,6 +113,7 @@ <h3>Multi-Post Stories</h3>
       </article>
 
       <!-- CARDS -->
```
Comment (KDlamini) [inline index.html:115]: The must be a clickable link on your README that has the URL of your deployed site.
If you do not know how to deploy your site. You can go to your repo, and click on `settings`:
image

Then go to `pages` > choose the branch you want to deploy > and save. You will be provided with the URL for your deployment. Copy the URL and add it to your README.
image


- If you don't know how to add the URL to your readme you can do something similar to this below your project description:
image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @dunnyk 🙋‍♀️,  

Good job so far! 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
 - [ ] Kindly add the URL of your site on your README.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dunnyk** [comment]
hi [KDlamini](https://github.com/KDlamini), I have done the changes asked. Kindly check.

**Review by gemmen29 — APPROVED** [review]
Hi @dunnyk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**